oak/server/oak_runtime.cc

```diff
@@ -118,7 +117,22 @@ void OakRuntime::SetUpChannels() {
     t.detach();
   }
 
-  // TODO: Set up remaining channels here rather than in OakNode.
+  // Create the channels needed for gRPC interactions.
+  {
```

Comment:
No longer a need for the explicit `{` `}` blocks now that the `shared_ptr<MessageChannel>` is being held on to.

Reply:
Done.